dspace-xmlui-mirage2/src/main/webapp/xsl/core/forms.xsl

```diff
@@ -265,6 +265,28 @@
     </xsl:template>
 
 
+    <xsl:template name="languageSelectBuilder">
+        <xsl:param name="name"/>
+        <xsl:param name="selected"/>
+        <select>
+            <xsl:call-template name="fieldAttributes"/>
+            <xsl:apply-templates/>
+            <xsl:attribute name="name"><xsl:value-of select="concat($name, '[lang]')"/></xsl:attribute>
+            <xsl:for-each select="../dri:params/dri:param[@name='langs']/dri:option | ./dri:params/dri:param[@name='langs']/dri:option">
```

Comment:
The `../dri:params` part is never used? As far as I can see, the template is only called with a `test="dri:params/dri:param[@name = 'langs']"`around. So the `dri:params` element should always be a child of the current element.

Reply:
I'll confess this and the dri2xhtml-alt code were directly ported from https://github.com/DSpace/DSpace/pull/1365, I'll try removing the `../dri:params` and will update the PR as needed.